doc/internal/ReleaseInstructions.md

```diff
@@ -95,137 +114,175 @@ Install `gpg-agent` (needed below) e.g. on Ubuntu via: `sudo apt-get install gnu
-    ```
-
-* As prompted in the `do_release` Makefile command's output, push the the `v12.0.0-rc1` tag.
-    ```shell
-    git push upstream v12.0.0-rc1
-    ```
-
-* Release the tag on GitHub UI as explained in the following section.
-
-------
-### Pre-Requisites for Releases
-
-> In this example our current version is `v11` and we release the version `v12.0.0`.
-
-
-* Announce dates on Vitess slack `#release-planning`.
-
-* Fetch `github.com/vitessio/vitess`'s remote.
-    ```shell
-    git fetch <vitessio/vitess remote>
-    ```
-
-* Build a Release Notes document using the Makefile:
-    ```shell
-    make VERSION="v12.0.0" FROM="<ref/SHA of the latest tag for v11>" TO="<ref/SHA of main>" SUMMARY="./doc/releasenotes/12_0_0_summary.md" release-notes
-    ```
+-------
 
-* Check to make sure all labels and categories set for each PR.
+## Release Cutover 
 
-    > The Release Notes document can be put aside for now, it will be used when creating the release tag in a future step.
+In this section we describe our current release process. We begin with a short [**overview**](#overview).
+The release process is divided into three parts: [**Pre-Release**](#pre-release), [**Release**](#release), [**Post-Release**](#post-release), which are detailed after the overview.
+
+### Overview
+
+#### Schedule
+
+A new major version of Vitess is released every four months. For each major version there is at least one release candidate, which we release three weeks before the GA version.
+We usually create the RC1 during the first week of the month, and the GA version three weeks later.
+
+#### Code Freeze
+
+Before creating RC1, there is a code freeze. Assuming the release of RC1 happens on a Tuesday, the release branch will be frozen Friday of the previous week.
+This allows us to test that the release branch can be released and avoid discovering unwanted events during the release day. Once the RC1 is released, there are three more weeks to backport bug fixes into the release branches. However, we also proceed to a code freeze the Friday before the GA release. (Assuming GA is on a Tuesday)
+Regarding patch releases, no code freeze is planned.
+
+#### Tracking Issue for each Release
+
+For each release, it is recommended to create an issue like [this one](https://github.com/vitessio/vitess/issues/10476) to track the current and past progress of a release.
+It also allows us to document what happened during a release.
+
+### Pre-Release
+
+This step happens a few weeks before the actual release (whether it is an RC, GA or a patch release).
+The main goal of this step is to make sure everything is ready to be released for the release day.
+That includes:
+- **Making sure Pull Requests are being reviewed and merged.**
+  > - All the Pull Requests that needs to be in the release must be reviewed and merged before the code freeze.
+  > - The code freeze usually happens a few days before the release. Make sure everything is merged by that date.
```

Comment:
This line can be deleted? It's basically the same as the previous line, right?

Reply:
Correct, I removed the last sentence as it is redundant.